src/metax_api/tests/api/base/apitests/middleware/test_middleware.py

```diff
@@ -125,3 +129,80 @@ def test_invalid_auth_method(self):
         self._use_http_authorization(header_value=b'NotSupported hubbabubba')
         response = self.client.post('/rest/datasets')
         self.assertEqual(response.status_code, FORBIDDEN)
+
+
+class ApiModifyResponseTestV1(APITestCase, TestClassUtils):
```

Comment:
setUp and getting test data looks very similar to CatalogRecordApiWriteCommon. Might want to import and inherit that instead of replicating the code.
from metax_api.tests.api.base.apitests.catalog_records.write import CatalogRecordApiWriteCommon

Reply:
Good point